Minor changes in the installation instructions for Ubuntu 14.04. Added the correct velocity limits for 800 model

rkojcev opened the pull request.

ahundt commented:
hey thanks for the patch! sorry for the problems, it seems some of the changes I just made don't mesh perfectly with yours. Could you perform a merge with the current master so I can accept your changes? Also boost 1.54 should now be ok

rkojcev commented:
@ahundt I have pulled the last version from your master and made couple of changes. Mainly I added the velocity limits for the 800 model and removed the secondsPerTick variable.
